src/test/java/com/google/devtools/build/lib/bazel/repository/starlark/BUILD

```diff
@@ -54,7 +56,6 @@ java_library(
 java_test(
     name = "StarlarkTests",
     testonly = 1,
-    srcs = glob(["*.java"]),
```

Comment:
Why removing this?

Reply:
It was duplicated with the `java_library` target and ended up compiling all files twice.